src/modules/twap/pure/PrimaryActionButton/index.tsx

```diff
@@ -19,22 +17,17 @@ const buttonsMap: Record<TwapFormState, (context: PrimaryActionButtonContext) =>
   ),
   [TwapFormState.NOT_SAFE]: () => (
     <ButtonPrimary disabled={true} buttonSize={ButtonSize.BIG}>
-      Please, connect to Safe
+      Unsupported wallet
     </ButtonPrimary>
   ),
-  [TwapFormState.ORDER_NOT_SPECIFIED]: () => (
+  [TwapFormState.NEED_FALLBACK_HANDLER]: () => (
     <ButtonPrimary disabled={true} buttonSize={ButtonSize.BIG}>
-      Please, specify an order
+      Unsupported Safe
```

Comment:
Too long if we write this?

```suggestion
      Unsupported Safe Fallback Handler
```

Reply:
Fair enough